vexim/siteaddsubmit.php

```diff
@@ -143,25 +139,10 @@
       die;
     }
   } else if ($_POST['type'] == "alias") {
-    $query = "SELECT domain_id FROM domains
```

Comment:
Before the changes, you had to type in the domain name of the destination domain.

Now, you can choose the destination domain from a list (https://github.com/Udera/vexim2/blob/undef/vexim/siteadd.php#L175-L184) and the `domain_id` is submitted.